R/utils.R

```diff
@@ -327,4 +335,66 @@ valid_seq <- function(params) {
   } else {
     return(FALSE)
   }
-}
\ No newline at end of file
+}
+
+resolve_prefix <- function(prefix, environment_var) {
+  if (!(is.null(prefix))) {
+    assertthat::assert_that(is.character(prefix))
+    
+    return(prefix)
+  } else {
+    prefix_env <- Sys.getenv(environment_var)
+    if (prefix_env != "") {
+      return(prefix_env)
+    } else {
+      return("/")
+    }
+  }
+}
+
+get_package_mapping <- function(script_path, url_package, dependencies) {
+  package_map <- vapply(dependencies, function(x) {
```

Comment:
I'm having a hard time following this function. Part of it is definitely my lack of R knowledge but it would be helpful if there were some comments that explained what these variables look like. For example:
```
# script_path is e.g. `/.../
# url_package is e.g. `...`
# dependencies is e.g. ...
# this function returns ...
```

Reply:
OK, will add.

Reply:
fixed in bea8ffc1f058dab55533a55a888d04bfb55144d0